README.md

```diff
@@ -64,7 +64,24 @@ To get the source code from GitHub, type
 1. Make sure you have installed **[Compoeser](https://getcomposer.org/doc/00-intro.md#introduction)** in your system.
```

Comment:
These lines obviously are checking the integrity of the installer, but the installer itself has a lot of other checks. You can take a look at the [composer repo](https://github.com/composer/getcomposer.org/blob/6504762e1d932f9098cc83688c6388f3d6451608/web/installer).
Whatever, downloading a file directly from a remote server without any check of integrity (like you are proposing) is very vulnerable for any kind of attack, like MITM by instance.
You can dive into this topic at [this issue](https://github.com/composer/composer/issues/1074) and take it just as an example.